src/class-elements/grammar-privatename-no-receiver-error-accessor-getter-meth.case

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: It is syntax error if IdentifierName is accessed without receiver (Accessor get Method)
```

Comment:
This description language is very similar to normative specification prose used to describe "Static Semantics: Early Errors"—however, I cannot find any such early error with this or similar language.